src/UI/Component/Item/Factory.php

```diff
@@ -0,0 +1,78 @@
+<?php
+namespace ILIAS\UI\Component\Listing;
+/**
+ * This is how a factory for Items looks like.
+ */
+interface Factory {
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      The Appointment Item is used to summarize one single
+	 *      appointment in a list.
```

Comment:
Or other contexts that use items, or not? Not sure if e.g. Decks become a valid context.